main.go

```diff
@@ -18,6 +18,10 @@ import (
 	"flag"
 	"os"
 
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
```

Comment:
Import order is odd. Make sure there's a linter for this.

Reply:
mmm this is really strange, I'm pretty sure I ran the linter before commit this. Will check again.